repondeur/zam_repondeur/views/amendements.py

```diff
@@ -39,6 +37,7 @@ def __init__(self, context: AmendementResource, request: Request) -> None:
         )
 
     @view_config(request_method="GET")
+    @view_config(request_method="GET", name="amendement_edit")  # backwards compat
```

Comment:
I don't get why we need backwards compatibility here and below?

Reply:
Changed urls → open pages before deployment may become 404s after